migrate/20250121_enable_transparent_cache_by_default.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:github_installation) do
+      set_column_default(:cache_enabled, true)
+    end
+
+    run "UPDATE github_installation SET cache_enabled = true"
```

Comment:
Should we enable this only for new customers for a while and observe their reactions? Should we update it for existing customers at this time?

Reply:
As we talked about sending informational email to existing customers in batch, updated PR to enable it only for new ones.